### PR TITLE
Add front-end support for sorting by `activity` and `creation date` in Search Page

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5375

⚠️ Requires: https://github.com/opencollective/opencollective-api/pull/7351

sorting-support

### CODE REVIEW

**Comment by znarf** [comment]
In all our recent talks, we never mentioned something like "Most recent activity", we're talking about a number that represents the volume of financial activity.

File: components/SearchForm.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ const SearchButton = styled(Flex)`
 class SearchForm extends React.Component {
   handleSubmit = event => {
     const searchInput = event.target.elements.q;
-    this.props.router.push({ pathname: '/search', query: { q: searchInput.value } });
+    this.props.router.push({ pathname: '/search', query: { q: searchInput.value, sortBy: 'ACTIVITY' } });
```
Comment (znarf) [inline components/SearchForm.js:44]: Is that really needed, can't we have this the default another way?

**Review by znarf — APPROVED** [review]
Looks good to me.

File: components/SearchForm.js (hunk 1 above)
Comment (SudharakaP) [reply components/SearchForm.js]: Only reason I added this was to add `sortBy="ACTIVITY"` to the query string in the url; but thinking further I think this isn't needed as whether we add that or not the sort by is defaulted to `ACTIVITY` anyways. Therefore I've removed this part.

**Pull request merged**